tasks/amplify.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Check if Amplify Agent is installed
+  package:
+    name: nginx-amplify-agent
+    state: present
+  ignore_errors: true
+  register: amplify_agent_installed
+
+- name: Install Amplify Agent if not installed
+  block:
+    - name: Download Amplify Agent script
+      get_url:
+        url: "{{ nginx_amplify_script_url }}"
+        dest: "{{ nginx_amplify_script_path }}"
```

Comment:
Part of installation should be tagged as "packages" and not "configuration"

Reply:
I've added the tags to the `block` rather than the `get_url` task.